.github/workflows/translations-sync.yml

```
@@ -0,0 +1,55 @@
+# This action automates the synchronization of our crowdin translations, so that a human does not need to kick it off from the crowdin UI
+# See translations-pr-lint-and-format.yml for quality control we conduct on ingress of new translations.
+name: Crowdin Sync
+
+on:
+  workflow_dispatch: # Allow running when we want to, for events such as urgent translation mistakes or 100% completed languages
+  schedule:
+    - cron: '0 5 * * 5' # At 05:00 on Fridays. This guarantees that we have the 72 hour weekend time to review translations.
+
+# Cancel any runs on the same branch
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  # These permissions required by `crowdin/github-action`
+  contents: write
+  pull-requests: write
+
+jobs:
+  synchronize-with-crowdin:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@91182cccc01eb5e619899d80e4e971d6181294a7 # v2.10.1
+        with:
+          egress-policy: audit
+
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+        # see all the options at https://github.com/crowdin/github-action
+      - name: crowdin action
+        uses: crowdin/github-action@2d540f18b0a416b1fbf2ee5be35841bd380fc1da #v2.3.0
+        with:
+          # testing only
+          dryrun_action: true
+          # do not upload existing translations - this is a one-way operation
+          upload_sources: true
+          upload_translations: false
+          # the rest of this controls how the PR comes in with new translations
+          download_translations: true
+          localization_branch_name: chore/crowdin
```

Maybe create a single branch for each workflow run?

```suggestion
          localization_branch_name: chore/crowdin-${{ github.sha }}
```

---

we never have more than one at a time